autobotAI_integrations/integrations/aws_bedrock/__init__.py

```
@@ -306,18 +318,22 @@ def _get_bedrock_model_request(
                 "temperature": float(temperature),
             }
         request = json.dumps(native_request)
-        return request    
+        return request
 
     def get_pydantic_agent(
         self, model: str, tools, system_prompt: str, options: dict = {}
```

Ruff (0.15.6)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)